`Container command '/bin/sh' not found or does not exist.`

Running `sudo make release=jessie` currently fails with the error below.

```
[…]
I: Base system installed successfully.
mkdir jessie-amd64
cd roots/jessie-amd64 \
        && tar -c --numeric-owner -f ../../jessie-amd64/root.tar ./
sed 's/SUBSTITUTION_FAILED/8b5a41e4d551e7024ea80902d305a31b39b07d96/' Dockerfile.in >jessie-amd64/Dockerfile
docker build --no-cache=true -t giantmonkey/debian:jessie-amd64 jessie-amd64
Sending build context to Docker daemon 227.8 MB
Step 1 : FROM scratch
 --->
Step 2 : MAINTAINER Jonathan Dowland <jmtd@debian.org>
 ---> Running in bbc133c4e531
 ---> 08825e9115a1
Removing intermediate container bbc133c4e531
Step 3 : ADD root.tar /
 ---> e5d4de8697f6
Removing intermediate container 4fb438285e3c
Step 4 : RUN if grep -q jessie /etc/apt/sources.list; then         echo "deb http://security.debian.org jessie/updates main contrib non-free"             >> /etc/apt/sources.list;     elif grep -q stretch /etc/apt/sources.list; then         echo "deb http://security.debian.org stretch/updates main contrib non-free"             >> /etc/apt/sources.list;     fi
 ---> Running in 7d6857c4c81c
Container command '/bin/sh' not found or does not exist.
Makefile:15: recipe for target 'build' failed
make: *** [build] Error 1
```

I got the same error when trying to build jessie. I tried running make clean and retrying but still got the error.

$sudo make release=jessie prefix=vincentlu arch=amd64 mirror=http://httpredir.debian.org/debian/

...
Step 4 : RUN if grep -q jessie /etc/apt/sources.list; then         echo "deb http://security.debian.org jessie/updates main contrib non-free"             >> /etc/apt/sources.list;     elif grep -q stretch /etc/apt/sources.list; then         echo "deb http://security.debian.org stretch/updates main contrib non-free"             >> /etc/apt/sources.list;     fi
 ---> Running in 9c4836e0ec6c
exec: "/bin/sh": stat /bin/sh: no such file or directory
Container command not found or does not exist.
Makefile:15: recipe for target 'build' failed
make: **\* [build] Error 1

--- a/Makefile
+++ b/Makefile
@@ -33,8 +33,8 @@ roots/$(tag):
 	mkdir -p $@
 
 roots/$(tag)/etc: roots/$(tag)
-	debootstrap --arch $(arch) $(release) $@ $(mirror) \
-		&& chroot $@ apt-get clean
+	debootstrap --arch $(arch) $(release) $< $(mirror) \
+		&& chroot $< apt-get clean
 
 clean:
 	rm -f $(tag)/root.tar $(tag)/Dockerfile

